firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigFetchHttpClient.java

```diff
@@ -210,15 +210,15 @@ FetchResponse fetch(
       }
     }
 
+    long endTime = System.currentTimeMillis();
```

Comment:
Can we add a corresponding test to https://github.com/firebase/firebase-android-sdk/blob/rc-telemetry/firebase-config/src/test/java/com/google/firebase/remoteconfig/internal/ConfigFetchHttpClientTest.java ? Thanks!

Reply:
Done and ran test successfully.